commit/report.go

```diff
@@ -104,3 +112,13 @@ func (p *Plugin) ShouldTransmitAcceptedReport(
 	)
 	return true, nil
 }
+
+func (p *Plugin) isGreenInstance(ctx context.Context) bool {
+	ocrConfigs, err := p.homeChain.GetOCRConfigs(ctx, p.donID, consts.PluginTypeCommit)
+	if err != nil {
+		p.lggr.Errorw("failed to get ocr configs from home chain", "err", err)
+		return false
```

Comment:
i think we need an error here, to distinguish between `isGreenInstance` and actual `error`. Because we might assume it's blue while it's green.